app/test_main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+import pytest
+
+from app.main import can_access_google_page
+
+from unittest import mock
+
+from typing import Callable
+
+
+@pytest.fixture()
+def mocked_valid_google_url() -> mock:
+    with mock.patch("app.main.valid_google_url") as mock_connection:
+        yield mock_connection
+
+
+@pytest.fixture()
+def mocked_has_internet_connection() -> mock:
+    with mock.patch("app.main.has_internet_connection") as mock_time:
+        yield mock_time
+
+
+def test_access_google_page_true(
+    mocked_valid_google_url: Callable,
+    mocked_has_internet_connection: Callable
+) -> None:
+    mocked_valid_google_url.return_value = True
+    mocked_has_internet_connection.return_value = True
+    assert can_access_google_page("www.google.com") == "Accessible"
```

Comment:
Add an error message. It's good when the user understands what the problem is. Fix everywhere

Reply:
Added

Reply:
You should add it here as well. And use more descriptive messages, for example, a web page should be Accessible if ....

Reply:
Messages changed